solvers/rslope.py

```diff
@@ -0,0 +1,77 @@
+from benchopt import BaseSolver, safe_import_context
+from benchopt.stopping_criterion import INFINITY, SufficientProgressCriterion
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from benchopt.helpers.r_lang import import_rpackages
+    from rpy2 import robjects
+    from rpy2.robjects import numpy2ri, packages
+    from scipy import sparse
+
+    # Setup the system to allow rpy2 running
+    numpy2ri.activate()
+    import_rpackages("SLOPE")
+
+
+class Solver(BaseSolver):
+    name = "rSLOPE"
+
+    install_cmd = "conda"
+    requirements = ["r-base", "rpy2", "r-slope", "r-matrix"]
+    references = [
+        "M. Bogdan, E. van den Berg, Chiara Sabatti, Weijie Su, and Emmanuel J.",
```

Comment:
Nitpick: some first names are not abbreviated. I'd abbreviate all.

Reply:
and  I think Emmanuel J. should be E. J. Candès also ;)